bedita-app/models/behaviors/build_filter.php

```diff
@@ -992,6 +992,9 @@ protected function user_createdFilter($s, $e) {
         $this->fields .= $locFields;
         $this->from .= ", {$s}users{$e} AS {$s}User{$e}";
         $this->conditions[] = "{$s}User{$e}.{$s}id{$e}={$s}BEObject{$e}.{$s}user_created{$e}";
+        if (!empty($this->filter['user_created'])) {
```

Comment:
the value of `$this->filter['user_created']` is passed directly to filter function. You can add a `$value` parameter to this method e use it
```php
protected function user_createdFilter($s, $e, $value) {
    ....
    if (!empty($value)) {
        $this->conditions[] = "{$s}BEObject{$e}.{$s}user_created{$e}={$value}";
    }
    ....
}
```